.github/workflows/build_and_test.yaml

```diff
@@ -69,11 +69,12 @@ jobs:
       - name: Build Player
         shell: bash
         run: |
+          cd build/
```

Comment:
All you need is `python build/all.py`.  (Better than build.py, IMO)  No need to change directories after actions/checkout with defaults.